src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -33,6 +33,11 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->defaultFalse()
                 ->end()
             ->end()
+            ->children()
+                ->variableNode('product_image_prefix')
+                    ->defaultValue('media/image')
```

Comment:
I'm fine with defining this value as a parameter, however we should at least use another parameter, that will be used here as well: https://github.com/Sylius/Sylius/blob/5dd517bb2198409d3f5723dfef1c290ec3f991e5/src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml#L60

But once we will introduce the parameter there, I'm not sure we need it. Up to you.